wemake_python_styleguide/visitors/tokenize/primitives.py

```diff
@@ -12,6 +13,9 @@
 
 class WrongPrimitivesVisitor(BaseTokenVisitor):
     """Visits primitive types to find incorrect usages."""
+    
+    # For consistency in usage octal, binary, and hex numbers
+    _bad_number_suffixes = 'XOBE'
```

Comment:
Can you please make it a tuple? This would be more readable. Currently it looks like a single word.